Common/Vulkan/VulkanImage.cpp

```diff
@@ -58,6 +58,12 @@ bool VulkanTexture::CreateDirect(VkCommandBuffer cmd, int w, int h, int numMips,
 		image_create_info.initialLayout = VK_IMAGE_LAYOUT_UNDEFINED;
 	}
 
+	// The graphics debugger always "needs" TRANSFER_SRC but in practice doesn't matter - 
+	// unless validation is on. So let's only force it on when being validated, for now.
+	if (vulkan_->GetFlags() & VULKAN_FLAG_VALIDATE) {
+		image_create_info.usage |= VK_IMAGE_USAGE_TRANSFER_SRC_BIT;
```

Comment:
Hm, dangerous, I wish this was done inside the validation layer so we could still have our usage flags validated... but I guess we're stuck with how it works.

-[Unknown]